Services/DeviceModels.cs

```diff
@@ -22,6 +22,9 @@ public interface IDeviceModels
 
     public class DeviceModels : IDeviceModels
     {
+        // ID used for custom device models, where the list of sensors is provided by the user
+        public const string CUSTOM_DEVICE_MODEL_ID = "custom";
```

Comment:
out of scope?  I like the change though- makes it more readable

Reply:
it was in scope at some point, to avoid duplicating the constant which was used here and in `SimulationRunner.cs`.  Eventually due to refactoring it's not strictly required, but I think the constant belongs here rather than in the runner.